threeSum.py

```diff
@@ -0,0 +1,23 @@
+# Time Complexity : O(N^2) for quick sort
```

Comment:
Sort the array then a for loop to get the first number then two pointers on the remaining array.